### DISCUSSION TITLE
how to use windows hello with 2fast

### DISCUSSION THREAD

**Discussion by DSI-Groupe-STURNO** [2fast-team/2fast#161] [Q&A] [answered]
Hello

I have enabled the option to use windows hello as login option, but the button "login with windows hello" keep greyed and the information tell me : "Function not available: Windows hello cannot be executed because no password is stored for the datafile in the protected memory".
What should I have to do to have this enabled ?

Best regards

**Comment by jp-weber** [comment, marked as answer]
Hello, I assume that the file was started with a double click? In this case, the app cannot be unlocked with windows hello because the app only saves a password in a protected memory and by starting the file with the application, the app does not access the protected memory.

if it is the same file that was used to create the app, the app must be started directly and not the file with a double click.

**Comment by DSI-Groupe-STURNO** [comment]
Hello,

Thanks for your quick reply.
According to your recommendation I confirm that windows hello works as soon as the app  is started by the app but not by the file.

May I add an additional information, that if you start the application by the file (double-click) then change its password, it then is impossible to start the app again other than  by the file. If you start it by the app, it doesn't longer recognize the new password as it requires the former one, and then is not able to open the file. The only solution then to recover such a situation, is to :
- open the app by the file again
- reset settings to factory defaults
- close the app, the open it by the app
- ask it to load the file. Once done you will be able again to open the app by the app and use windows hello

Best regards

**Reply by jp-weber** [reply]
A new version is now available as release candidate, which resolve the problem of changing the password:
https://github.com/2fast-team/2fast/releases/tag/v1.3.1-rc2

**Comment by jp-weber** [comment]
Thank you for the feedback, I can reproduce the problem and will fix it in the next few days with version 1.3.1.
